app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -278,6 +287,11 @@ export default {
         clearTimeout(this.idleTimer);
       }
     },
+    handleEnterKeyWhenEnterToSendEnabled(event) {
```

Comment:
Found a bug on the help center article editor - https://www.loom.com/share/11f005fa91144b8cb9e089d7c38e4b37